src/components/screens/vacation-requests-screen.tsx

```diff
@@ -424,6 +476,9 @@ const VacationRequestsScreen = () => {
           updateVacationRequestStatuses={updateVacationRequestStatuses}
           loading={loading}
         />
+        {renderVacationDays(
+          persons.find((person) => person.id === loggedInPerson?.id || config.person.id)
```

Comment:
As mentioned in the vacations card could you not use the loggedInPerson here rather than the fine. Also some padding or similar needs to be added to the content returned from renderVacationDays here as it looks a little tightly packed in at the bottom of the table.